packages/wiz/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.1"
```

```suggestion
- version: "1.7.0"
```
since this is an enhancement.